### PR TITLE
Term entry c sharp cosh

### LINKED ISSUE #1765: [Term Entry] C# Math Functions `.Cosh()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `Math.Cosh()` method.

The entry should go in a new file in `docs/content/c-sharp/concepts/math-functions/terms/cosh/cosh.md`

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for C# Math Functions: Math.Cosh()
- Closes #1765 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/cosh/cosh.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Math.Cosh(angle);
+```
+
+The `Math.Cosh()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the hyperbolic cosine of the `angle` as a `double` value, except if the value of `angle` equals:
+
+- `NaN` (not a number), then it returns `NaN`
+- `NegativeInfinity`, then it returns `PositiveInfinity`
+- `PositiveInfinity`, then it also returns `PositiveInfinity`
+
+> **Note:** Depending on the used operating system or architecture, the exact result or the input range of the `Math.Cosh()` method may differ due to the differences in the C runtime environment.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/cosh/cosh.md:31]:
```suggestion
> **Note:** Depending on the operating system or architecture, the exact result or the input range of the `Math.Cosh()` method may differ due to the differences in the C runtime environment.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a first review and left one comment.

**Comment by cslylla** [comment]
@SSwiniarski Thank you, I accepted it!

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good for review two.

**Review by yangc95 — APPROVED** [review]

**Pull request merged**